qa/tasks/vstart_runner.py

```diff
@@ -909,7 +909,7 @@ def clear_firewall(self):
         # FIXME: unimplemented
         pass
 
-    def newfs(self, name='cephfs', create=True):
+    def newfs(self, name='testcephfs1', create=True):
```

Comment:
Let's match the behavior of `newfs` in `MDSCluster`.

Reply:
Done.